src/pages/Cart.css

```diff
@@ -0,0 +1,183 @@
+:root {
+    --cart-card-background-color: #082032;
+    --cart-card-aliceblue-color: aliceblue;
+    --cart-card-a-hover-color: blueviolet;
+    --cart-card-white-color: white;
+    --cart-card-black-color: black;
+  }
+  .card {
+    box-shadow: 0 0 10px 10px var(--cart-card-aliceblue-color);
+    max-width: 300px;
+    margin: auto;
+    text-align: center;
+    font-family: arial;
+    background-color: var(--cart-card-aliceblue-color);
+  }
+  
+  .price {
+    color: grey;
+    font-size: 22px;
+  }
+  
+  .card button {
+    border: none;
+    outline: 0;
+    padding-left: 2rem;
+    padding-right: 2rem;
+    padding-top: 11.2px;
+    padding-bottom: 8px;
+    color: var(--cart-card-white-color);
+    background-color: var(--cart-card-background-color);
+    text-align: center;
+    cursor: pointer;
+    width: 100%;
+    font-size: 18px;
+    position: relative;
+    font-weight: 800;
+    top: 2rem;
+  }
+  .card button {
+    text-decoration: none;
+    border-radius: 5px;
+  }
+  .card button:hover {
+    color: var(--cart-card-a-hover-color);
+    opacity: 1;
+    border: none;
+    background-color: var(--primary-color);
+  }
+  .cards {
+    display: flex;
+    flex-direction: row;
+    gap: 1rem;
+    justify-content: space-evenly;
+    text-align: center;
+    width: 100%;
+    flex-wrap: wrap-reverse;
+    z-index: 0;
+    position: absolute;
+    top: 10rem;
+  }
+  .card-btn-add {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+  }
+  .card {
+    width: 40%;
+  }
+  .quantity-up-down {
+    margin-top: 4.8px;
+    display: flex;
+    flex-direction: row;
```

Comment:
Suggestion: You are using some css property repeatedly, instead of this you can make common class for them and use them where they are needed